packages/letters-of-credit/src/components/LoCCard/LoCCard.js

```diff
@@ -48,19 +49,34 @@ class LoCCard extends Component {
   }
 
   generateCardContents(letter, user) {
-    let contents = (
-      <div>
-        <h3>{'Ref: ' + letter.letterId}</h3>
-        <p>{'Participants: Alice, ' + letter.issuingBank + ', Bob, ' + letter.exportingBank}</p>
-        <p>{'Product Type: ' + letter.productDetails.productType}</p>
-        <button className="viewButton" onClick={() => this.handleOnClick()}>
-          <div className = "viewButtonImage">
-            <img src = {viewButtonIcon} alt = ""/>
+    let contents;
```

Comment:
A couple of things here that just need updating to properly match the designs:

Whenever a new letter appears that Bob hasn't approved yet, it should say "NEW" at the top - you should be able to check this easily enough by seeing if 'bob' is in the approval array of the letter. Cards he hasn't yet approved should look like this:

<img width="225" alt="screen shot 2018-04-27 at 14 01 37" src="https://user-images.githubusercontent.com/35696678/39363811-855dba56-4a23-11e8-9ace-c30618452cf3.png">


Whenever Bob ships a letter the arrow to view it is still there, and the background of the card gets filled in like this:

<img width="213" alt="screen shot 2018-04-27 at 13 54 47" src="https://user-images.githubusercontent.com/35696678/39363528-9237c7d6-4a22-11e8-9f54-830f0ff1522b.png">

Alice should also still be able to view the letter when it is time for her to accept the goods - her cards should look like this: 

<img width="307" alt="screen shot 2018-04-27 at 13 59 21" src="https://user-images.githubusercontent.com/35696678/39363731-3ddc60e2-4a23-11e8-8980-49d2b146367e.png">